api/resources_portal/views/report_issue.py

```diff
@@ -1,6 +1,7 @@
 import json
 
 from django.conf import settings
+from django.http import JsonResponse
 from rest_framework import status
 from rest_framework.decorators import api_view, permission_classes
 from rest_framework.permissions import IsAuthenticated
```

Comment:
In line 36, it's mentioned that this is an issue with a resource request. Here, I use the endpoint to file an issue with the grants. Should that text be configurable, or am I missing something?

Reply:
I was missing something. Nevermind,